airbyte-integrations/connectors/source-hubspot/unit_tests/integrations/test_contacts_form_submissions.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+
+import freezegun
+from airbyte_cdk.test.mock_http import HttpMocker
+from airbyte_protocol.models import AirbyteStateBlob, AirbyteStateMessage, AirbyteStateType, AirbyteStreamState, StreamDescriptor, SyncMode
+
+from . import HubspotContactsTestCase
+from .request_builders.streams import ContactsStreamRequestBuilder
+
+
+@freezegun.freeze_time("2024-05-04T00:00:00Z")
```

Comment:
It's better to use the dynamic `end_date` freeze, like `pendulum.parse(<config.start_date>).substract(days=<x_days_to_substract>)`, having the 1 `start_date` anchor, instead of hardcoding the value.

Reply:
Also, these tests are similar except of the `self.mock_response` and `assert output.state_messages`, etc assertions, which make me think we can simplify the logic of these tests having 1 executable test_method, and bunch of inputs / outputs (parametrized). DYT it worth making so?